tests/unit/lms/resources/lti_launch_test.py

```diff
@@ -104,9 +148,9 @@ def JSConfig(patch):
 @pytest.fixture
 def pyramid_request(pyramid_request):
     pyramid_request.parsed_params = pyramid_request.lti_params = {
-        "tool_consumer_instance_guid": mock.sentinel.tool_guid,
-        "resource_link_id": mock.sentinel.resource_link_id,
-        "context_id": mock.sentinel.context_id,
-        "context_title": mock.sentinel.context_title,
+        "tool_consumer_instance_guid": sentinel.tool_guid,
+        "resource_link_id": sentinel.resource_link_id,
+        "context_id": sentinel.context_id,
+        "context_title": sentinel.context_title,
```

Comment:
:+1: It's ok to import things. Let's treat ourselves.

Seriously though, we have enough eye clutter in general, changes like this lower the cognitive load to read the file and make this easier to maintain. We use `sentinel` everywhere, it's not ambiguous what we mean.

Reply:
Random observation, this is specific to the `TestGroupingType` test and should probably be inlined there. Try commenting it out and running the tests again.